src/utils/ChoreUtils.jsx

```diff
@@ -0,0 +1,19 @@
+// Helper function used in ChoreItem.jsx to calculate a new date. 
+// Used to find a new due date and reminder date for a reoccurring chore.
+export function getNextDate(current, frequency) {
+  const newDueDate = new Date(current);
+  if (frequency === "daily") newDueDate.setDate(newDueDate.getDate() + 1);
+  if (frequency === "weekly") newDueDate.setDate(newDueDate.getDate() + 7);
+  if (frequency === "monthly") newDueDate.setDate(newDueDate.getDate() + 28);
+  return newDueDate;
+}
+
+// Helper function used in ChoreItem.jsx to assign new assignee for a chore item.
+export function getNextAssignee(currentAssignee) {
+  const assigneesList = ["Leeza", "Amanda", "Josh", "Sesame"];
```

Comment:
I'm not sure this allows for chores that can only be assigned to specific people because it would always rotate within this list. But what if a chore only has "Leeza" as the possible assignee? It would still get "Amanda" as the next assignee.

Reply:
I wanted to implement a simple assignee rotation feature for now as sort of a placeholder. I was planning to go back and refine it when I implement a family/group page feature later.